help_popup/static/src/css/popup_help.css

```diff
@@ -0,0 +1,12 @@
+.breadcrumb_hide {
+    width: 0%;
+}
+
+.breadcrumb_show {
+    width: 44% !important;
```

Comment:
Never use `!important` unless someone's life depends on it, please 😊 

Besides, not much sense on these classes... Elements should be *shown* by default, unless explicitly *hidden*, for which you already have the `.hidden` bootstrap class.